vowpalwabbit/core/src/reductions/epsilon_decay.cc

```diff
@@ -455,7 +455,7 @@ std::shared_ptr<VW::LEARNER::learner> VW::reductions::epsilon_decay_setup(VW::se
                  .set_input_label_type(VW::label_type_t::CB)
                  .set_output_label_type(VW::label_type_t::CB)
                  .set_input_prediction_type(VW::prediction_type_t::ACTION_PROBS)
-                 .set_output_prediction_type(VW::prediction_type_t::ACTION_SCORES)
+                 .set_output_prediction_type(VW::prediction_type_t::ACTION_PROBS)
```

Comment:
maybe add a runtest with edecay and explore eval?

Reply:
yea good idea